backends/fs/stdiostream.h

```diff
@@ -27,20 +27,35 @@
 #include "common/stream.h"
 #include "common/str.h"
 
+#if defined(__DC__)
+// libronin doesn't support rename
```

Comment:
Perhaps in such cases, we could read the contents of the temp file in memory, and dump them in the target file? This would simulate a rename

Reply:
In fact, libronin doesn't support writing files at all.

The problem when building comes from the fact that libronin provides read, write calls but not rename.
Then, the linker uses the newlib rename function which, in its default implementation, reads the old file, writes to the new file and deletes the old one afterwards. But it is using the newlib functions and there is a collision when linking.
That's why I avoid using the rename function on DC.

Reply:
Thanks for the explanation :)